src/components/Commander.jsx

```diff
@@ -1,15 +1,22 @@
 import './Commander.scss'
 import React from 'react'
+import os from 'socket:os'
+import { useSelector } from 'react-redux'
 
 // Components
 import { PanelView, CommandBar, Taskbar } from './index'
+import { MobilePanelView, MobileCommandBar } from './mobile/index'
 
 const Commander = () => {
+  const { isMobile } = useSelector((state) => state.mobilePlatforms)
+
   return (
-    <main className="commander">
-      <CommandBar />
-      <PanelView />
-      <Taskbar />
+    <main className={isMobile ? 'mobile-commander' : 'commander'}>
+      {!isMobile && os.platform() === 'win32' && <CommandBar />}
+      {isMobile && <MobileCommandBar />}
+      {!isMobile && os.platform() === 'win32' && <PanelView />}
+      {isMobile && <MobilePanelView />}
+      {!isMobile && os.platform() === 'win32' && <Taskbar />}
```

Comment:
What is the reasoning behind testing for win32 specifically? Have you considered the other desktop platforms (darwin, linux)?

Reply:
This is just a temporary solution. 'win32' represents the desktop version of our application, but of course we want our app to be available on other systems as well. This solution is just for the developers' purposes. Unless you have a better solution in mind that we can use, please let me know. We were talking about implementing a system detection which will replace this temporary solution.